diff -uraN tagpy-2018.1.1/src/wrapper/id3.cpp omv-tagpy-2018.1.1/src/wrapper/id3.cpp
--- tagpy-2018.1.1/src/wrapper/id3.cpp	2013-07-10 13:56:25.000000000 +0200
+++ omv-tagpy-2018.1.1/src/wrapper/id3.cpp	2020-12-31 13:22:49.898778705 +0100
@@ -220,12 +220,7 @@
       .DEF_SIMPLE_METHOD(removeFrame)
       .DEF_SIMPLE_METHOD(removeFrames)
 
-      #if (TAGPY_TAGLIB_HEX_VERSION >= 0x10800)
-        .DEF_OVERLOADED_METHOD(render, ByteVector (cl::*)() const)
-        .DEF_OVERLOADED_METHOD(render, ByteVector (cl::*)(int) const)
-      #else
-        .def("render", (ByteVector (cl::*)() const) &cl::render)
-      #endif
+      .def("render", (ByteVector (cl::*)() const) &cl::render)
       ;
   }
 
